docs/elements.md

```diff
@@ -12,95 +12,68 @@ documentation.
-  a group.
-- [`pl-matrix-component-input`](#pl-matrix-component-input-element): Fill in
-  a **matrix** using grid that has an input area for each element.
-- [`pl-matrix-input`](#pl-matrix-input-element): Supply a matrix in a supported
-  programming language format.
-- [`pl-multiple-choice`](#pl-multiple-choice-element): Selecting only
-  **one option** from a list.
-- [`pl-number-input`](#pl-number-input-element): Fill in a **numerical** value
-  within a specific tolerance level such as 3.14, -1.921, and so on.
+- [`pl-integer-input`](#pl-integer-input-element): Fill in an **integer** value such as -71, 0, 5, 21, and so on.
+- [`pl-matching`](#pl-matching-element): Select a matching option for each entry in a group.
+- [`pl-matrix-component-input`](#pl-matrix-component-input-element): Fill in a **matrix** using grid that has an input area for each element.
+- [`pl-matrix-input`](#pl-matrix-input-element): Supply a matrix in a supported programming language format.
+- [`pl-multiple-choice`](#pl-multiple-choice-element): Selecting only **one option** from a list.
+- [`pl-number-input`](#pl-number-input-element): Fill in a **numerical** value within a specific tolerance level such as 3.14, -1.921, and so on.
 - [`pl-order-blocks`](#pl-order-blocks-element): Select and arrange given blocks of code or text.
-- [`pl-rich-text-editor`](#pl-rich-text-editor-element): Provide an in-browser formattable text editor
-  for writing and submitting code.
-- [`pl-string-input`](#pl-string-input-element): Fill in a **string** value
-  such as `"Illinois"`, `"GATTACA"`, `"computer"`, and so on.
-- [`pl-symbolic-input`](#pl-symbolic-input-element): Fill in a **symbolic** value
-  such as `x^2`, `sin(z)`, `mc^2`, and so on.
-- [`pl-units-input`](#pl-units-input-element): Fill in a **number** and a **unit**
-  such as "1.5 m", "14 ms", "6.3 ft", and so on.
+- [`pl-rich-text-editor`](#pl-rich-text-editor-element): Provide an in-browser formattable text editor for writing and submitting prose.
+- [`pl-string-input`](#pl-string-input-element): Fill in a **string** value such as `"Illinois"`, `"GATTACA"`, `"computer"`, and so on.
+- [`pl-symbolic-input`](#pl-symbolic-input-element): Fill in a **symbolic** value such as `x^2`, `sin(z)`, `mc^2`, and so on.
+- [`pl-units-input`](#pl-units-input-element): Fill in a **number** and a **unit** such as "1.5 m", "14 ms", "6.3 ft", and so on.
 
 **Decorative** elements are meant to improve how the question is displayed to
 students. Elements under this category include ways to specify question markup,
 images, files, and code display. The following **decorative** elements are available:
 
 - [`pl-card`](#pl-card-element): Displays content within a card-styled component.
-- [`pl-code`](#pl-code-element): Displays code rendered with the appropriate
-  syntax highlighting.
+- [`pl-code`](#pl-code-element): Displays code rendered with the appropriate syntax highlighting.
```

Comment:
Same comment as above, here and in the lines below, though these don't bother me as much...